maintainers.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: simple-page
+title: RIOT Maintainers
+subtitle: RIOT is nurtured by a strong community of maintainers.
+---
+<main>
+  <!-- ======= Maintainers Section ======= -->
+  <section id="maintainers">
+    <div class="container">
+      <p>
+      This is the current list of maintainers within the RIOT community with there area of
+      expertise within the
+      <a href="https://github.com/RIOT-OS/RIOT/" target="_blank">RIOT codebase</a>.
+
+      If a maintainer is marked with <q>Has no chosen area of expertise</q>, they did not add any
+      ownership within
+      <a href="https://github.com/RIOT-OS/RIOT/blob/master/CODEOWNERS" target="_blank">
+        <tt>CODEOWNERS</tt>
+      </a>. This does not mean that they do not feel responsible for any part
+      of the code base, they just did not declare it.
+      </p>
+
+      <p id="maintainer-list">
```

Comment:
a (responsive) grid layout seems better than showing elements in a list. the current list shows a lot of whitespace since each card doesn't include long descriptions.

Reply:
:heavy_check_mark:

Reply:
The full list now looks like this

![image](https://github.com/user-attachments/assets/4dcbbcc0-77cd-49fc-9bf7-2f2acc240372)

Reply:
a bit imbalanced with respect to the whitespace of each card, but i think it's ok. @leandrolanzieri any opinion?

Reply:
Since the selection of patterns is imbalanced, I don't think there is a way around it, unless we sort the mainatainers by their list length of patterns... which seems weird.

Reply:
Maybe in this case we could try something like a masonry layout instead? https://getbootstrap.com/docs/5.0/examples/masonry/

Reply:
This is how it would look like. Less negative space, but IMHO the order is now a bit intransparent

![image](https://github.com/user-attachments/assets/0664c78a-1e55-467d-8981-bcd925042134)

Reply:
I see that we have an alphabetical order, but I don't think that's particularly important, right?

I like this last version much more!

Reply:
Any reason for names to be on a new line now?

Reply:
Applied.

Reply:
> Any reason for names to be on a new line now?

Since some names wrapped, when they + the nickname were longer, I decided to generally put them on the next line. Looks better this way, in my opinion.

Reply:
> Since some names wrapped, when they + the nickname were longer, I decided to generally put them on the next line. Looks better this way, in my opinion.

See https://github.com/RIOT-OS/riot-os.org/pull/141/commits/fa2fcb38d5ca99bff69e09424e1cd5b45aab4eca